server/views/temporary-accommodation/premises/index.njk

```diff
@@ -59,10 +59,6 @@
 
 {% endblock %}
 
- {% block extraScripts %}
-     <script type="text/javascript" nonce="{{ cspNonce }}">
-       $(function() {
-         window.MOJFrontend.initAll()
-       })
-     </script>
- {% endblock %}
+{% block extraScripts %}
+    <script src="/assets/js/jquery.min.js"></script>
```

Comment:
This would have been better to setup a separate commit or comment to explain that this only temporary until we move away from moj sortable table component